lib/rust/ensogl/core/src/display/camera/camera2d.rs

```diff
@@ -406,6 +406,7 @@ impl Camera2d {
         self.data.borrow_mut().update(scene)
     }
 
+    // FIXME: this can fail - updated to shared registry already.
```

Comment:
Fixme left

Reply:
I improved the comment. Will be fixed in another PR. This is not a new thing and this PR is already huge.